demos/client-example/tsconfig.json

```diff
@@ -10,15 +10,9 @@
         {
             "path": "../../packages/procaptcha-react"
         },
-        {
```

Comment:
Why is this removed? Procaptcha frictionless is still in the package

Reply:
I have done